apps/frontend/src/app/alert-page/component/alert-page/alert-page.component.html

```diff
@@ -0,0 +1,149 @@
+            </p>
+            <div
+              *ngIf="summary.repeatedAlerts.length > 1"
+              class="repeated-alerts"
+            >
+              <small>Other repeated alerts:</small>
+              <ul class="list-unstyled">
+                <li *ngFor="let alert of summary.repeatedAlerts.slice(1, 4)">
+                  {{ alert.type }}: {{ alert.count }} times
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- Error Message -->
+  <clr-alert *ngIf="error" class="alert-danger">
+    <clr-alert-item>
+      <span class="alert-text">{{ error }}</span>
+    </clr-alert-item>
+  </clr-alert>
+
+  <!-- Alert Grid -->
+  <clr-datagrid [clrDgLoading]="loading" (clrDgRefresh)="refresh($event)">
+    <clr-dg-column>Severity</clr-dg-column>
+    <clr-dg-column>Date</clr-dg-column>
+    <clr-dg-column>Type</clr-dg-column>
+    <clr-dg-column>Details</clr-dg-column>
+
+    <clr-dg-row *ngFor="let alert of alerts$ | async" [clrDgItem]="alert">
+      <clr-dg-cell>
+        <div
+          class="severity-badge"
+          [ngClass]="{
+            'severity-critical':
+              alert.alertType.severity === SeverityType.CRITICAL,
+            'severity-warning':
+              alert.alertType.severity === SeverityType.WARNING,
+            'severity-info': alert.alertType.severity === SeverityType.INFO
+          }"
+        >
+          <cds-icon
+            [attr.shape]="
+              alert.alertType.severity === SeverityType.CRITICAL
+                ? 'error-standard'
+                : alert.alertType.severity === SeverityType.WARNING
+                ? 'warning-standard'
+                : 'info-standard'
+            "
+            class="is-solid"
+          >
+          </cds-icon>
+          {{ alert.alertType.severity }}
+        </div>
+      </clr-dg-cell>
+      <clr-dg-cell>{{ formatDate(alert.creationDate) }}</clr-dg-cell>
+      <clr-dg-cell>{{ alert.alertType.name }}</clr-dg-cell>
+      <clr-dg-cell>
+        <span style="font-weight: bold">{{ getAlertReason(alert) }}</span>
+        <ng-container [ngSwitch]="alert.alertType.name">
+          <div *ngSwitchCase="'SIZE_ALERT'">
+            <div *ngIf="hasBackup(alert)">
+              <strong>Backup Saveset:</strong> {{ alert.backup.saveset }}<br />
+              <strong>Creation Date:</strong>
+              {{ formatDate(alert.backup.creationDate) }}
+            </div>
+            {{ getAlertDetails(alert) }}
+          </div>
+
+          <div *ngSwitchCase="'STORAGE_FILL_ALERT'">
+            <ng-container *ngIf="isStorageFillAlert(alert)">
+              <strong>Storage Fill:</strong>
+              {{ shortenBytes(alert.filled * 1024 * 1024 * 1024) }} /
+              {{ shortenBytes(alert.capacity * 1024 * 1024 * 1024) }}<br />
+              <strong>High Water Mark:</strong>
+              {{ shortenBytes(alert.highWaterMark * 1024 * 1024 * 1024) }}
```

Comment:
```suggestion
              {{ shortenBytes(alert.filled * 1000 * 1000 * 1000) }} /
              {{ shortenBytes(alert.capacity * 1000 * 1000 * 1000) }}<br />
              <strong>High Water Mark:</strong>
              {{ shortenBytes(alert.highWaterMark * 1000 * 1000 * 1000) }}
```
Metric is GB, not GiB